site/versioning-policy.html

```diff
@@ -251,30 +251,30 @@ <h2>Release Cadence</h2>
 generally be released about 6 months after 2.2.0. Maintenance releases happen as needed
 in between feature releases. Major releases do not happen according to a fixed schedule.</p>
 
-<h3>Spark 2.4 Release Window</h3>
+<h3>Spark 3.0 Release Window</h3>
 
 <table>
   <thead>
     <tr>
       <th>Date</th>
-      <th>&#160;</th>
       <th>Event</th>
     </tr>
   </thead>
   <tbody>
     <tr>
-      <td>Mid Aug 2018</td>
-      <td>&#160;</td>
+      <td>Late Oct 2019</td>
+      <td>Preview release</td>
+    </tr>
+    <tr>
+      <td>Early Dec 2019</td>
       <td>Code freeze. Release branch cut.</td>
     </tr>
     <tr>
-      <td>Late Aug 2018</td>
-      <td>&#160;</td>
+      <td>Late Dec 2019</td>
       <td>QA period. Focus on bug fixes, tests, stability and docs. Generally, no new features merged.</td>
     </tr>
     <tr>
-      <td>Early Sep 2018</td>
-      <td>&#160;</td>
+      <td>Jan 2019</td>
```

Comment:
Oops. Sorry, I missed this. This should be `2020`.

Reply:
Ack, oops. I'm sure people will get the idea but feel free to hotfix it or whatever

Reply:
Sure. I'll make a PR for this.